vars/buildUnityWdkV2.groovy

```diff
@@ -79,14 +82,18 @@ def call(Map configMap = [unityVersions: []]) {
               }
 
               cleanup {
-                cleanWs()
+                script {
+                  if(config.mainPlatform.clearWs) {
+                    cleanWs()
+                  }
+                }
               }
             }
           }
 
           stage("setup upm") {
             agent {
-              label "atlas && $config.buildLabel"
+              label "atlas && macos"
```

Comment:
Just curious, why did we decide `macos` is now a hard-coded requirement as opposed to before being a configurable setting ?

Reply:
Because those steps are indiferent from the platform that the tests are executed. As we now execute the test/build in multiple platforms, we cant associate it to a single variable anymore. MacOS is the safest option because the pipeline run the most on it, so I chose to hardcode it for now.